plugins/sudo/sudo.plugin.sh

```diff
@@ -1,11 +1,21 @@
 #! bash oh-my-bash.module
 
 function _omb_plugin_sudo_add_sudo {
+  function __omb_plugin_sudo_is_inserted {
+    [[ "${READLINE_LINE::4}" = "sudo" ]]; return $?
```

Comment:
```suggestion
    [[ $READLINE_LINE == sudo* ]]
```

- There is no need to explicitly write `return $?`. The exit status of the function is that of the last command executed in the function.
- Also, we can actually use the glob pattern matching here.
- Another thing I thought is that maybe there is a command name like `sudoxxxx` in which case we might still prefix `sudo ` when <kbd>ESC ESC</kbd> is pressed. But probably we do not need to prefix `sudo ` for such a command because such a command would probably call `sudo` inside like `sudoedit`. But I'm not sure if we should switch the behavior depending on the detailed context or knowledge about the commands because it will make it cognitively harder for humans to predict what happens next by pressing each key.

Reply:
I don't like to use glob pattern matching, to fix the problem presented I can get the 5th character and check if it's a space.

Would I use glob pattern matching anyway? If yes, it will still with that problem

Reply:
> Would I use glob pattern matching anyway? If yes, it will still with that problem

The glob pattern can include a space.

```suggestion
    [[ $READLINE_LINE == 'sudo '* ]]
```

Reply:
Ok, I'll make use of glob pattern